tools/identity-resolution/identity-resolution/Models/AadTokenCredential.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Azure.Sdk.Tools.NotificationConfiguration.Models
+{
+    /// <summary>
+    /// This the client to generate the aad auth token.
+    /// </summary>
+    public class AadTokenCredential : IDisposable
```

Comment:
Any reason we need this type? Can we not use https://docs.microsoft.com/en-us/dotnet/api/azure.identity.clientsecretcredential?view=azure-dotnet?

Reply:
Tried in local, I encountered a conflict between Azure.Identity and [Microsoft.VisualStudio.Services.Identity](https://github.com/sima-zhu/azure-sdk-tools/blob/msAlias/tools/identity-resolution/identity-resolution/Services/AzureDevOpsService.cs#L14).

Reply:
Can you give me more details about this conflict? Is it because they have conflicting types or is there some other conflict?

Reply:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/48036328/153250332-b2bab2de-1d7e-4beb-8233-d9a67c7f9f8d.png">

Run `dotnet build`

Here is the error message:
```
C:\sdk\azure-sdk-tools\tools\identity-resolution\identity-resolution\Services\AzureDevOpsService.cs(258,27): error CS0118: 'Identity' is a namespace but is used like a type [C:\sdk\azure-sdk-tools\tools\identity-resolution\identity-resolution\identity-resolution.csproj]
    0 Warning(s)
    1 Error(s)
```

Reply:
It works. Thanks for pointing me to the right direction.